### PR TITLE
Enhance UI elements and add OpenSource section on homepage

### PR DESCRIPTION
**Pull request by davidsmonkeys**
## Description

There are a couple of changes in this PR that support the launch of DERT2025. These are: 

- Provide additional validation on the entering of scores for the individual judges as the border was too small. 
- Set the registration open now sticker 
- Remove debugging information from the email templates screen
- Move the help section on th ehomepage so it's above the page fold
- Provide informative information on the CSV downloads

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New features (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Validated on local

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules### :bulb: PR Summary generated by FirstMate

**Overview**: Enhanced UI elements and added an OpenSource section on the homepage.

**Changes**:
*UI Enhancements:*
- Increased border thickness for `.scores-valid` class in `split-scores-grid.component.css`.
- Added informational text below the CSV download button in `dances-grid.component.html`.

*Homepage Updates:*
- Removed unnecessary text from the Email Templates header in `event-email-templates.component.html`.
- Introduced a new OpenSource section in `home.component.html` with details and a GitHub link.

**TLDR**: Focus on the new OpenSource section added to the homepage and UI enhancements for better user experience.

<sub><sup>Generated by FirstMate and automatically updated on every commit.</sup></sub>

### CODE REVIEW

File: src/client/src/app/modules/event-admin/components/event-email-templates/event-email-templates.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div id="app-addition-header">
     <mat-card class="mat-bg-default">
       <mat-card-title>
-        <mat-icon>email</mat-icon>Email Templates X {{isReady}} Y {{showEditor}} Z
+        <mat-icon>email</mat-icon>Email Templates <!--X {{isReady}} Y {{showEditor}} Z -->
       </mat-card-title>
```
[the remark this answers is not in the document]
Comment (davidsmonkeys) [reply src/client/src/app/modules/event-admin/components/event-email-templates/event-email-templates.component.html]: Resolves and removed comment as no value to commenting out the debugging. It can be added back in if required.

**Review by dertinfo — APPROVED** [review]
